core/trino-parser/src/test/java/io/trino/sql/parser/TestSqlParser.java

```diff
@@ -349,13 +350,8 @@ public void testArraySubscript()
         assertExpression("ARRAY [1, 2][1]", new SubscriptExpression(
                 new ArrayConstructor(ImmutableList.of(new LongLiteral("1"), new LongLiteral("2"))),
                 new LongLiteral("1")));
-        try {
-            assertExpression("CASE WHEN TRUE THEN ARRAY[1,2] END[1]", null);
-            fail();
-        }
-        catch (RuntimeException e) {
-            // Expected
-        }
+        assertThatThrownBy(() -> assertExpression("CASE WHEN TRUE THEN ARRAY[1,2] END[1]", null))
+                .isInstanceOf(NullPointerException.class);
     }
```

Comment:
shall we also validate a message?

Reply:
@MariyaR It seems that there is no message for this exception.